<?xml version="1.0"?>
<launch>

  <arg name="map_file_name" />
  <arg name="lat_origin" />
  <arg name="lon_origin" />

  <param name="/lanelet2_interface_ros/map_file_name" value="$(arg map_file_name)" />
  <param name="/lanelet2_interface_ros/map_frame_id" value="map" />
  <param name="/lanelet2_interface_ros/map_with_offset_frame_id" value="map_with_offset" />
  <param name="/lanelet2_interface_ros/lat_origin" value="$(arg lat_origin)" />
  <param name="/lanelet2_interface_ros/lon_origin" value="$(arg lon_origin)" />

  <node pkg="lanelet2_interface_ros" type="map_to_mapWithOffset_tf_publisher.py" name="map_to_mapWithOffset_tf_publisher" output="screen" />

</launch>

